upup/pkg/fi/README.md

```diff
@@ -0,0 +1,30 @@
+# fi - Fast Install
+
+The `fi` package is responsible for holding the infrastructure that supports installing, provisioning and deploying resources to get kubernetes running in a cloud.
```

Comment:
Lets keep it here for now - I think having docs in the code is a much broader conversation. At least now we can't point users to a github URL that will have some more detail on the mysterious `fi` directory.